cmd/brew-test-bot.rb

```diff
@@ -1103,6 +1103,10 @@ def cleanup_shared
       Dir.glob("#{HOMEBREW_PREFIX}/**/*").each do |path|
         next if path.start_with?(HOMEBREW_REPOSITORY.to_s)
         next if prefix_paths_to_keep.include?(path)
+        # don't try to delete osxfuse files
+        next if path.match?(
+          "(include|lib)/(lib|osxfuse/|pkgconfig/)?(osx|mac)?fuse(.*\.(dylib|h|la|pc))?$",
```

Comment:
It appears that on the CI machines, the whole `/usr/local/lib/pkgconfig/` directory was deleted.

```
$ ls -lh /usr/local/lib
total 4200
lrwxr-xr-x  1 brewadmin  staff    13B Sep 25 15:45 libfuse.0.dylib -> libfuse.dylib
-rwxr-xr-x  1 brewadmin  staff   582K Jul  1 18:02 libfuse.2.dylib
lrwxr-xr-x  1 brewadmin  staff    15B Sep 25 15:45 libfuse.dylib -> libfuse.2.dylib
-rwxr-xr-x  1 brewadmin  staff   917B Jul  1 18:02 libfuse.la
-rwxr-xr-x  1 brewadmin  staff   583K Jul  1 18:02 libfuse_ino64.2.dylib
lrwxr-xr-x  1 brewadmin  staff    21B Sep 25 15:45 libfuse_ino64.dylib -> libfuse_ino64.2.dylib
-rwxr-xr-x  1 brewadmin  staff   947B Jul  1 18:02 libfuse_ino64.la
-rwxr-xr-x  1 brewadmin  staff   918K Jul  1 18:01 libosxfuse.2.dylib
lrwxr-xr-x  1 brewadmin  staff    18B Sep 25 15:45 libosxfuse.dylib -> libosxfuse.2.dylib
-rwxr-xr-x  1 brewadmin  staff   933B Jul  1 18:01 libosxfuse.la
lrwxr-xr-x  1 brewadmin  staff    18B Sep 25 15:45 libosxfuse_i64.2.dylib -> libosxfuse.2.dylib
lrwxr-xr-x  1 brewadmin  staff    16B Sep 25 15:45 libosxfuse_i64.dylib -> libosxfuse.dylib
lrwxr-xr-x  1 brewadmin  staff    13B Sep 25 15:45 libosxfuse_i64.la -> libosxfuse.la
```